Source/ThirdParty/easy_profiler/CMakeLists.txt

```diff
@@ -25,7 +27,13 @@ macro(easy_define_target_option TARGET SOURCE_OPTION TARGET_DEFINITION)
 endmacro()
 
 add_subdirectory(easy_profiler_core)
-add_subdirectory(profiler_gui)
+# ATOMIC BEGIN
+if (MSVC AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+    message(STATUS "Profiler GUI can only be built in Release builds with MSVC compiler.")
```

Comment:
I do not think it is possible to kick certain targets into "Release" build when everything else is building as "Debug". But maybe we could just remove `-DDEBUG=1` (which is implicit) and use `/MD` and have release build within debug build. That that might work, i will try.